### PR TITLE
[Botskills] Replace botbuilder-tools to botframework-cli tools

### PR DESCRIPTION
**Pull request by Batta32**
### Purpose
*What is the context of this pull request? Why is it being done?*
As [botbuilder-tools](https://github.com/Microsoft/botbuilder-tools) will be deprecated, it's necessary to migrate the internal tools (Ludown. LuisGen) to the ones of [botframework-cli](https://github.com/microsoft/botframework-cli)

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Replace `ludown` to `bf luis:convert` command
- Replace `luisgen` to `bf luis:generate:[cs|ts]`
- Update documentation
- Update botskills tests
- Fix `ESLint` warnings

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
\-

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\-

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [X] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

File: docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ toc: true
 ## Prerequisites
 
 - [Node.js](https://nodejs.org/) version 10.8 or higher
-- Install the Dispatch, LUDown and LUISGen CLI tools
+- Install the Dispatch, BF-CLI, BF-LUIS-CLI and LUISGen CLI tools
 
     ```shell
-    npm install -g botdispatch ludown luisgen
+    npm install -g botdispatch @microsoft/botframework-cli @microsoft/bf-luis-cli luisgen
```
Comment (darrenj) [inline docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md]: To the best of my knowledge we just need to install **just** botframework-cli as this incorporates all the LUIS CLI capabilities?

Comment (darrenj) [inline docs/_docs/skills/handbook/add-skills-to-a-virtual-assistant.md]: We can also remove luisgen at this stage as bf luis:generate does this for you :-)

File: tools/botskills/test/connect.test.js [hunk 2]
```diff
@@ -343,13 +343,13 @@ Error: Path to the nonExistenceen-usDispatch.dispatch file leads to a nonexisten
 
             strictEqual(errorList[errorList.length - 1], `There was an error while connecting the Skill to the Assistant:
 Error: An error ocurred while updating the Dispatch model:
-Error: There was an error in the ludown parse command:
-Command: ludown parse toluis --in "${join(configuration.luisFolder, configuration.languages[0], "connectableSkill.lu")}" --luis_culture ${configuration.languages[0]} --out_folder "${join(configuration.luisFolder, configuration.languages[0])}" --out connectableSkill.luis
+Error: There was an error in the bf luis:convert command:
```
Comment (darrenj) [inline tools/botskills/test/connect.test.js:346]: Could we test the scenario whereby the user has not followed the pre-reqs and installed botframework-cli - if so, show a message guiding to install this? Covers the scenarios where people update and don't read the docs!

**Review by darrenj — CHANGES_REQUESTED** [review]
See below - we only need the one CLI now. If changing luisgen at this time to luis:generate is too complex we can take this PR/change now and followup with that later!

**Comment by Batta32** [comment]
@darrenj - we will check to migrate `luisgen` to `luisgen:generate` and we will get back to you later 😊!

File: tools/botskills/test/connect.test.js (hunk 2 above)
Comment (Batta32) [reply tools/botskills/test/connect.test.js:346]: Sure! We will verify the scenario you mentioned.
Do you agree to apply those changes in another PR just to improve the error's messages?

**Comment by Batta32** [comment]
@darrenj - we already migrated `luisgen` to `bf luis:generate:[cs|ts]` 😊!

File: tools/botskills/test/connect.test.js (hunk 2 above)
Comment (darrenj) [reply tools/botskills/test/connect.test.js:346]: Yes that works

**Comment by darrenj** [comment]
re: luisgen that's great but we should remove from doc steps!

**Comment by Batta32** [comment]
@darrenj - we removed `luisgen` and `ludown` from the botskills documentation as you can see in the changes. Did we miss something?

**Review by darrenj — APPROVED** [review]

**Review by ryanisgrig — APPROVED** [review]

**Pull request merged**